charts/mungegithub/chart/values.kubernetes.kubernetes.yaml

```diff
@@ -12,11 +12,9 @@ config:
   http-cache-dir: /cache/httpcache
   organization: kubernetes
   project: kubernetes
-  # Make sure approval-handler and blunderbuss run before submit-queue.
+  # Make sure blunderbuss runs before submit-queue.
   # Otherwise it's going to take an extra-cycle to detect the label change.
-  # Run blunderbuss before approval-handler, so that we can suggest approvers
-  # based on assigned reviewer.
-  pr-mungers: blunderbuss,approval-handler,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,path-label,size,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue,issue-creator,triage-filer,flakyjob-reporter,old-test-getter,docs-need-no-retest,sig-mention-handler
+  pr-mungers: blunderbuss,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,path-label,size,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue,issue-creator,triage-filer,flakyjob-reporter,old-test-getter,docs-need-no-retest,sig-mention-handler
```

Comment:
this is out of date relative to the actual configmap being used, this was pre misc-mungers/submit-queue split... maybe these should be removed? for now my goal was just to remove the approval-handler reference